### ISSUE TITLE
how to export tensorrt int8  model with savedmodel format after calibration？

### ISSUE THREAD

**Issue by rankeey** [tensorflow/tensorflow#25378]
Is it possible to export a savedModel int8 model after calibration?
If it is possible, is there any demo could be offered？
If it is impossible，when will it maybe possible？

**Comment by trevor-m** [comment]
You can save the frozen graph that convert_after_calib returns and use it like a normal frozen graph. Not sure about how SavedModel works.

**Comment by trevor-m** [comment]
Yup, it is a normal frozen model. The only difference is that some of the nodes will be TRTEngineOp.

**Comment by rankeey** [comment]
@trevor-m 
I do conversion with the script from [stackflow](https://stackoverflow.com/questions/44329185/convert-a-graph-proto-pb-pbtxt-to-a-savedmodel-for-use-in-tensorflow-serving-o/44329200#44329200).
I found that the `logits` tensor lose shape info. And the export model could run neither in TFTIS nor in tf-serving.

the detail info you can reference to [#issue 25417](https://github.com/tensorflow/tensorflow/issues/25417)

The error message from TRTIS is as bellow:
` E0202 01:40:52.288811 1 aspired_versions_manager.cc:358] Servable {name: resnet_v1_50_graphdef version: 1} cannot be loaded: Invalid argument: unable to load model 'resnet_v1_50_graphdef', output 'logits' dims [] don't match configuration dims [1001]`

Loading the model with tf-serving is OK, but when do request for test, the server will crashed with a Segmentation fault.

**Comment by ymodak** [comment]
This question is better asked on [StackOverflow](http://stackoverflow.com/questions/tagged/tensorflow-tensorflow) since it is not a bug or feature request. There is also a larger community that reads questions there.

**Comment by rankeey** [comment]
@ymodak it's really a bug as i mentioned before, losing shape while do tensorrt transformation.

**Comment by ymodak** [comment]
@samikama Can you PTAL? Thanks!

**Comment by samikama** [comment]
@trevor-m @pooyadavoodi  Can you PTAL

**Comment by samikama** [comment]
@pooyadavoodi @trevor-m What is the status on here?
